versioned/tiered/rocksdb/src/test/java/com/dremio/nessie/versioned/store/rocksdb/TestRocksDBStore.java

```diff
@@ -48,13 +48,29 @@ static void tearDown() throws IOException {
    */
   @Override
   protected RocksDBStore createStore() {
-    return new RocksDBStore(DB_PATH.toString());
+    return new RocksDBStore(createConfig());
+  }
+
+  private RocksDBStoreConfig createConfig() {
```

Comment:
Rather than have two config creation functions, why not have the createConfig take the path to be returned?

Reply:
Done